_emails/fremont-ca.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Fremont, California
+permalink: "/fremont-ca"
+name: Letter to the Mayor, City Council, City Manager, and Police Chief
+city: Fremont
+state: CA
+recipients:
+- lmei@fremont.gov
+- councilmemberjones@fremont.gov
+- vbacon@fremont.gov
+- rsalwan@fremont.gov
+- tkeng@fremont.gov
+- jkassan@fremont.gov
+- yshao@fremont.gov
+- cof@fremont.gov
+- PoliceChief@fremont.gov
+subject: Fremont Resident for Defunding the Fremont PD
+body: |-
+  To Mayor Mei, the Fremont City Council, Fremont City Manager, and Fremont Police Chief
+
+  My name is \[YOUR NAME\], and I am a resident of Fremont. I am writing to demand that the City Council adopts a People’s Budget that prioritizes community wellbeing and redirects funding away from the police.
+  
+  This past week, our nation has been gripped by protests calling for rapid and meaningful change with regard to police behavior, an end to racism and anti-blackness, and immediate reform in how black people are treated in America. The Bay Area has been at the forefront of much of this action. Accordingly, it has come to my attention that the budget for 2021 is being decided as these protests continue.
+
+  Fremont PD has been a waste of our resources. Last year, the Fremont PD budget was $93,307,000, almost all of which comes from the Fremont general fund. This means Fremont PD cost us taxpayers almost 3 times as much as was spent on public street, facilities, and park maintenance, and human services combined. Fremont's projected 2020-21 budget appears to be heading in an extraordinarily similar direction.
+  
+  While we've been spending astonishing amounts of money on policing, we have not seen improvements to safety, homelessness, mental health, or affordability in our city. Instead, I see the wasteful and harmful actions of our police.
+
+  I join the calls of those across the country to #DefundThePolice. I demand a budget that adequately and effectively meets the needs of at-risk Fremont residents during this trying and uncertain time, when livelihoods are on the line. I call on you to slash the Fremont PD budget and instead use those resources towards solving homelessness. We demand a budget that supports community wellbeing, rather than empowers the police forces that tear them apart.
+  
+  As the City Council, the budget proposal is in your hands. It is your duty to represent your constituents. I am urging you to completely revise the budget for the 2020-2021 fiscal year, and to fund #CareNotCops.
+
+  We can be a beacon for other cities to follow if only we have the courage to change.
+
+  Sincerely,
+
+  \[YOUR NAME\]
+
```

Comment:
nit) the spaces in between these tags is not necessary.

Also, why are there escape characters? Shouldn't it just be

[YOUR NAME]
[YOUR ADDRESS]
etc.

?

Reply:
fixed!

No reason for the escape characters—copied from another template 😬